yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
@bimalgrg519, we don't need this file.

Reply:
@bimalgrg519 can you take a look at the comments when you have time, please?

Reply:
@cre8ivejp I mistakenly added this file. I have removed it now.